### ISSUE TITLE
[Feature Request] Flash region on screenshot

### ISSUE THREAD

**Issue by MithicSpirit** [naelstrof/maim#221]
It would be nice if there was a switch that could be set which would make the screenshotted region flash when the screenshot was taken so that I have some feedback that it actually worked. This wouldn't work with just executing another program that flashes the screen after maim because it should flash the specific region that was screenshotted (rather than the entire screen).

**Comment by foxpy** [comment]
Combine `slop` and `maim` like this:

```bash
flash() {
	echo "Replace this function with actual flash program which would flash screen at $1"
}

coord="$(slop)"
save_path="$HOME/screenshot_$(date).png"
maim --geometry="$coord" > "$save_path"
flash "$coord"
```

No someone should implement the `flash` program :)

**Comment by foxpy** [comment]
Yeah, `slop` is a part of `maim`, if you use `maim`, you almost definitely have `slop` installed.

**Comment by alvv-z** [comment]
In a script, you used to be able to call gnome's flash like this IIRC:
```bash 
gdbus call -e -d org.gnome.Shell -o /org/gnome/Shell/Screenshot -m org.gnome.Shell.Screenshot.FlashArea $ORIGX $ORIGY $WIDTH $HEIGHT
```

It no longer works since they've restricted the API access, but maybe someone with more knowledge could port [their flash mechanic](https://gitlab.gnome.org/GNOME/gnome-screenshot/-/blob/master/src/cheese-flash.c) to maim.
